cocos2d/Platforms/Android/CCGLView.h

```diff
@@ -41,9 +41,8 @@ enum CCAndroidScreenMode {
     CCScreenScaledAspectFitEmulationMode
 };
 
-BRIDGE_CLASS("com.apportable.GLView")
-@interface CCGLView : GLView <CCDirectorView>
-
+BRIDGE_CLASS("org.cocos2d.CCGLView")
```

Comment:
Not sure if those changes were merged into this branch. (There are too many to keep track of right now...)